demo/lightning/image-segmentation/data.py

```diff
@@ -62,9 +62,10 @@ def train_dataloader(self):
             batch_size=self.args.batch_size,
             shuffle=not self.args.benchmark and self.train_sampler is None,
             sampler=self.train_sampler,
-            num_workers=0,
+            num_workers=self.args.num_workers,
             pin_memory=True,
             drop_last=True,
+            prefetch_factor=100,
```

Comment:
It would probably be a good idea to make this configurable with a flag too

Reply:
Done!